webapp/channels/src/components/user_settings/display/__snapshots__/user_settings_display.test.tsx.snap

```diff
@@ -1237,7 +1237,7 @@ exports[`components/user_settings/display/UserSettingsDisplay should match snaps
                         type="radio"
                       />
                       <Memo(MemoizedFormattedMessage)
-                        defaultMessage="On"
+                        defaultMessage="Expanded"
```

Comment:
A couple of these had incorrect default messages in the code, so I changed them to match the ones in `en.json`